specifyweb/frontend/js_src/lib/components/WebLinks/spec.ts

```diff
@@ -220,6 +201,7 @@ function reconstructWeblink(
               name: localized(argument.name),
               title: localized(argument.title),
               legacyIsEditable: false,
+              shouldPrompt: undefined,
```

Comment:
can you please double check how would the XML for the "prompt" tag in the weblink look like after this change?
i.e did every weblink before this have a `<prompt>false</prompt>` or `<prompt>true</prompt>` tag?

if so, would a new web-link created by specify have `<prompt>false</prompt>` (correct) or not have `<prompt>` at all (potentially breaking sp6?)